### DISCUSSION TITLE
Add a link to the docs somewhere for easy access

### DISCUSSION THREAD

**Discussion by exi66** [Dokploy/dokploy#490] [General]
### What problem will this feature address?
Better quality of life. There is no link to the project itself anywhere and it is not very convenient to open documentation for every little thing through the address bar

### Describe the solution you'd like
Link somewhere in the header\footer\left menu navigation. Any option will do. Examples: 
![image](https://github.com/user-attachments/assets/33cc92fb-4a5a-4e06-a097-cbe2e5491456)
![image](https://github.com/user-attachments/assets/6fb949ce-06e4-4ecd-b25a-8176c83b8d9f)

**Comment by predragnikolic** [comment]
What is the use case? 
So you have some examples(designs) from other sites?
What would be the benefits?

**Comment by predragnikolic** [comment]
For me one of the huge benefits I see with Dokploy over completing software like Coolify, is that Dokploy has a cleaner UI. Adding new UI elements contribute to a bloated UI.

**Comment by exi66** [comment]
> What is the use case?

Dokploy is primarily a development tool and I would like to have quick access to the description of the tool's functionality: product documentation, api swagger etc. 

> So you have some examples(designs) from other sites?

The home page https://dokploy.com/ has a nice design and the header has all these links. No reference is needed - it already exists.

**Comment by predragnikolic** [comment]
Browsers already have a bookmark functionality.
Browser usually have a shortcut ctrl+t(or cmd+t on Mac) to open a new tab and focus the address bar, you can type docs dokploy, if you ever visited the dokploy docs the browser will probably suggest you the links that you visited.
If you do not want to open a new tab, you can focus the address bar with `ctrl+l` (or `cmd+l`)

> [!NOTE] 
> These shortcuts are for Chorome, but other browsers like Firefox, Arc also have similar keyboard shortcuts.

---
I'm pretty sure you knew all of the stuff I mention above.
The reason for why I'm writing this is that the thing that you want can be achieved currently with what you probably have.

**Comment by exi66** [comment]
You wrote that it would damage the clean design of the application, but is it bad to have the two links I described in the header, where navigation is supposed to be? Especially since the UI on the home page, in my opinion, is no less clean, although these links are there.

**Comment by predragnikolic** [comment]
I try to give a different point out of view for the thing that is requested.

I speak my mind as anyone on GitHub can speak their mind.
When you add something it is hard to remove it, because people get use to it.

> Especially since the UI on the home page, in my opinion, is no less clean, although these links are there.

The purpose of those two application is different. 
The home page is to give info to the users of Dokploy, so it makes sense that the site has links in the top nav.
The web app is to actually do deployment stuff. For that it makes sense for the top nav to contain features related to that.

Sometimes you just need to see UX from other web apps to see the solution people come up with this. They usually put docs, and such links in the footer.

<img width="1426" alt="Screenshot 2024-08-09 at 14 18 56" src="https://github.com/user-attachments/assets/9450447f-baa5-4ee8-bc72-ee9a9681a7ae">

<img width="1441" alt="Screenshot 2024-08-09 at 14 17 44" src="https://github.com/user-attachments/assets/0c99bf41-3054-48bc-9819-e2eb9f7a9541">

**Comment by exi66** [comment]
It can be placed in the footer, I wrote about this in the initial post. I only suggested what is easiest to implement - add a link to the header, since it already exists, it already has an example of a link and this is the default navigation. I don't care where it will be, I'm not a ux expert, the main function is quick access to the description of the tool to solve the problem yourself. It can also be placed in the left menu, for example: 
![image](https://github.com/user-attachments/assets/6fb949ce-06e4-4ecd-b25a-8176c83b8d9f)

I'm changing the title and the initial post so there's no more confusion about where to place the link

**Comment by b3nab** [comment]
Honestly I don't think this is really an issue since there are already tons of ways to open a link to a website (shortcuts, macros, bookmarks, simply open a new tab and write the link, having a list of useful links on some notes app like notion/evernote/obsidian or other methods).

But I'm converting it to discussion so you are free to continue this conversation also having all the ux/ui considerations. :)

**Reply by johngrimsey** [reply]
I'll +1 on this. This would be useful. Using Dokploy requires frequent references to docs. You might reduce the number of needless GitHub tickets :)

Put a small docs button in the top right next to support? It's a secondary navigation area already. I would argue that a docs button is a strong candidate for inclusion in this area.

ps - the Dokploy UI is _very_ good. Well done all 👍
